streams/src/main/java/org/apache/kafka/streams/processor/internals/ActiveTaskCreator.java

```diff
@@ -119,7 +119,8 @@ private Producer<byte[], byte[]> producer() {
     }
 
     public void reInitializeProducer() {
-        streamsProducer.resetProducer(producer());
+        if (!streamsProducer.isResetDisabled())
```

Comment:
nit (code style): we always use `{ }` for all blocks in KS code base

Reply:
Thanks for the reminder.
I will follow it.